apps/api/src/app/shared/framework/idempotency.interceptor.ts

```diff
@@ -34,6 +34,7 @@ enum ReqStatusEnum {
 export class IdempotencyInterceptor implements NestInterceptor {
   constructor(private readonly cacheService: CacheService) {}
 
+  @Instrument()
```

Comment:
Instrumenting the IdempotencyInterceptor for creating Idempotency Observability Dashboards.